README.md

```diff
@@ -73,32 +73,53 @@ However, care should be taken here because if the schema has been migrated in th
 See [here](docs/DisasterRecovery.md)
 
 ## Passwords
+
+Database passwords are managed by the vault and laid out ascii
```

Comment:
Why does this say "ascii"?

Reply:
Emacs has an out of control autocompletion feature that occasionally does this